style.css

```diff
@@ -0,0 +1,9 @@
+body  {
+background-color: aqua;
+}
+h1   {
+color: red;
+}
+p    {
+color: blue;
+}  
```

Comment:
Add configuration to your editor so it automatically adds new line at the end of file.
If you're using vscode, add `"files.insertFinalNewline": true` to the configuration.